packages/core/src/components/dropdown/dropdown.tsx

```diff
@@ -454,11 +429,37 @@ export class TdsDropdown {
   };
 
   private setValueAttribute = () => {
-    if (!this.value || this.value?.toString() === '') {
+    if (this.selectedOptions.length === 0) {
       this.host.removeAttribute('value');
     } else {
-      this.host.setAttribute('value', convertArrayToStrings(this.value).join(','));
+      this.host.setAttribute('value', this.selectedOptions.join(','));
+    }
+  };
+
+  getOpenDirection = () => {
```

Comment:
Maybe make this one private? 
```suggestion
  private getOpenDirection = () => {
```

Reply:
Good cathc!